circleci/submodule-github-release

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Publishes content from [MODULE_PATH] as a Github Release.
+# Repo must have a VERSION file in its base directory.
```

Comment:
While you're here, can you add to this file and the main script the expected format of VERSION (i.e. first line is the version, then blank line, then release notes)